File: LeetCode Algorithms/122.买卖股票的最佳时机-ii.py
#
# @lc app=leetcode.cn id=122 lang=python3
#
# [122] 买卖股票的最佳时机 II
#

# @lc code=start
class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        profit = 0
        for _index in range(1, len(prices)):
            tmp = prices[_index] - prices[_index - 1]
            if tmp > 0: profit += tmp
        return profit
# @lc code=end

File: LeetCode Algorithms/122.买卖股票的最佳时机-ii.cpp
/*
 * @lc app=leetcode.cn id=122 lang=cpp
 *
 * [122] 买卖股票的最佳时机 II
 */

// @lc code=start
class Solution {
public:
    int maxProfit(vector<int>& prices) {
        int ans = 0, tmp = 0;
        for (int _index = 1; _index < prices.size(); ++_index) {
            tmp = prices[_index] - prices[_index - 1];
        }
    }
};
// @lc code=end

